src/formatDate.js

```diff
@@ -50,7 +50,29 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const oldSeparator = fromFormat[3];
+  const newSeparator = toFormat[3];
+  const year = fromFormat.find(el => el.includes('Y'));
+  const yearIndex = fromFormat.indexOf(year);
+
+  const oldDate = date.split(oldSeparator);
+  const newDate = [];
+
+  for (let i = 0; i < toFormat.length - 1; i++) {
+    if (!fromFormat.includes(toFormat[i])
+      && fromFormat.includes('YY')
+      && toFormat.includes('YYYY')) {
```

Comment:
але краще буде винести в змінні

```suggestion
    if (
      !fromFormat.includes(toFormat[i])
      && fromFormat.includes('YY')
      && toFormat.includes('YYYY')
    ) {
```